src/index.js

```diff
@@ -0,0 +1,63 @@
+import './style.css';
+import {
+  FnToDoList,
+  addTask,
+  editTask,
+  deleteTask,
+} from './modules/UI.js';
+import clearCompleted from './modules/clearCompleted.js';
+import markTask from './modules/marktask.js';
+import { updateLocalStorage, getLocalStorage } from './modules/storage.js';
+
+const input = document.querySelector('.input');
+const List = document.querySelector('.list-ul');
+const clearCompletedBtn = document.querySelector('.clear-btn');
+const addTaskBtn = document.querySelector('.add-btn');
+
+let Tasks = getLocalStorage();
+
+FnToDoList(Tasks);
+
+input.addEventListener('keypress', (e) => {
+  if (e.key === 'Enter' && input.value !== '') {
+    addTask(Tasks, input.value);
+    input.value = '';
+    updateLocalStorage(Tasks);
+    FnToDoList(Tasks);
+  }
+});
```

Comment:
[] Kindly use proper descriptive names for naming your variables and functions, this is good practice for ease of readability.

Reply:
@SaneMyburg Thanks for your suggestion. 🤝